.github/workflows/01-build-and-release.yml

```diff
@@ -0,0 +1,138 @@
+# With thanks to https://gist.github.com/weibeld/f136048d0a82aacc063f42e684e3c494
+name: build-and-release
+on:
+  push:
+    branches:
+      - main
+permissions:
+  contents: write
+jobs:
+  release-job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          path: src
+      - name: Install DJGPP
+        run: |
+          pushd /opt
+          wget https://github.com/andrewwutw/build-djgpp/releases/download/v3.3/djgpp-linux64-gcc1210.tar.bz2
+          tar -xf djgpp-linux64-gcc1210.tar.bz2
+          rm djgpp-linux64-gcc1210.tar.bz2
+          popd
+      - name: Build project
+        run: |
+          cd $GITHUB_WORKSPACE/src
+          ls -lh main.c
+          . /opt/djgpp/setenv
+          make
+          ls -lh output/sbemu.exe
+      - name: Build FreeDOS SBEMU USB image
+        run: |
+          cd /tmp
+          wget https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/distributions/1.3/official/FD13-LiteUSB.zip
+          wget https://www.freedos.org/download/verify.txt
+          grep -q "64a934585087ccd91a18c55e20ee01f5f6762be712eeaa5f456be543778f9f7e  FD13-LiteUSB.zip" verify.txt
+          echo "64a934585087ccd91a18c55e20ee01f5f6762be712eeaa5f456be543778f9f7e  FD13-LiteUSB.zip" | shasum -a 256 --check
```

Comment:
What's the point of downloading verify.txt if the only thing we're doing with it is checking the hash contained within the file, only to then do the check without verify.txt:

```
echo "64a934585087ccd91a18c55e20ee01f5f6762be712eeaa5f456be543778f9f7e  FD13-LiteUSB.zip" | shasum -a 256 --check
```

I'd just remove the downloading and checking of `verify.txt`, as checking the hash of `FD13-LiteUSB.zip` is the important piece, and that's already done here (as it should be).

Also, if we do check the hash of the file (which is a good thing), why don't we check the hash of `djgpp-linux64-gcc1210.tar.bz2` downloaded above?